templates/default/vlan/vlanlist.html

```diff
@@ -64,5 +64,14 @@ <h1>{$pagetitle|escape}</h1>
 
 {include file="vlan/vlanmodify-dialog.html"}
 
+<script>
+    $( '.delete-vlan' ).click(function() {
+        confirmDialog( $t("Are you sure, you want to delete that VLAN?") , this).done(function() {
```

Comment:
Wiesz, że to wymaga przegenerowania strings.js w js/locales?

Reply:
Nie - rozumiem, że ten plik jest generowany z strings.php. Wrzucę tłumaczenie więc również tam.

Reply:
Proszę fb5de4fbc38c18661fb4c71e6069ef4d17bd18e3.

Reply:
Zawsze wrzucasz do string.php, a na jego podstawie przegenerowujesz js/...../strings.js.